$NetBSD$

* Avoid to call call __SQLGetInfo() twice since __SQLGetInfo() calls
  thread_release() internally and it cause unlocking unlocked mutex.

--- DriverManager/SQLConnect.c.orig	2008-09-29 23:02:43.000000000 +0900
+++ DriverManager/SQLConnect.c
@@ -2086,11 +2086,13 @@ int __connect_part_two( DMHDBC connectio
         char txt[ 20 ];
         SQLRETURN ret;
 
+#if 0
         ret = __SQLGetInfo( connection,
                     SQL_DRIVER_ODBC_VER,
                     txt,
                     sizeof( txt ),
                     NULL );
+#endif
 
         if ( SQL_SUCCEEDED( ret ))
         {
